Moderation-Policy.md

```diff
@@ -0,0 +1,139 @@
+* Any Collaborator who habitually authors Posts that must be Moderated can be Banned from further participation in the Node.js GitHub organization for an indefinite period of time. Such action can only be taken through normal TSC motion and vote (see: [Escalation to the TSC](#escalation-to-the-tsc)).
+
+#### Non-Collaborator Posts
+
+* Posts authored by non-Collaborators are always subject to immediate Moderation by any Collaborator if the content is intentionally disruptive or in violation of the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md).
+* When Moderating non-Collaborator Posts, the moderating Collaborator should:
+ * Explain the justification for Moderating the post, and
+ * Identify all changes made to the Post.
+* If an explanation of a Moderation action for a non-Collaborator Post is provided, it should be provided in:
+ * The original Post being modified (as replacement or appended content),
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Moderation of Posts authored by non-Collaborators may result in those non-Collaborators being Banned from further participation in the Node.js GitHub organization for an indefinite period of time.
+
+Note that Moderating non-Collaborator posts can often lead to retaliation or escalation of inappropriate behavior by the individual whose post is being Moderated. This is true primarily of individuals whose intent is to harass, disrupt or annoy individual members of the community. In such cases, it is best to handle the Moderation as quickly and as quietly as possible without drawing any further undue attention to the Post in question.
+
+#### Trolling
+
+Trolling is generally defined as the act of creating deliberately offensive or provocative Posts for the sole purpose of derailing a conversation, stealing attention, inciting an angry response or disrupting the usability of physical and electronic spaces used by the contributors to the Node.js project. More specifically, within the context of the Node.js project, Trolling consists of *any behavior considered by the TSC to be intentionally disruptive, offensive or abusive*.
+
+Trolling behavior of any kind is not tolerated and will be subject to immediate Moderation. The author of Trolling Posts will be subject to permanent Banning from the GitHub organization and will be recommended for exclusion from all Node.js Foundation events and activities.
+
+Determination of what constitutes "Trolling behavior" within the Node.js project is solely at the discretion of the TSC.
+
+### Escalation to the TSC
+
+Any Moderation issue or dispute can be escalated to the TSC by "mentioning" `@nodejs/tsc` in the body of a GitHub issue either in the original thread or in the private nodejs/moderation repository. Optionally, the `tsc-agenda` label may be attached to the issue to request that it be added to the TSC meeting agenda.
```

Comment:
At mentions of groups only work when you're a member of the node.js organization, I think.  If the intent of this section is that only collaborators escalate, that should be mentioned explicitly because it isn't clear from context (to me anyway.)